### DISCUSSION TITLE
Questions about owstaxidmetadataalltagsinfo and other managed properties

### DISCUSSION THREAD

**Discussion by gitulm** [microsoft-search/pnp-modern-search#3751] [Q&A]
I need help evaluating owstaxidmetadataalltagsinfo in the Results web part. If I display the tag owstaxidmetadataalltagsinfo in the result web part, the following appears:

![image](https://github.com/microsoft-search/pnp-modern-search/assets/56668705/4770e086-30e2-47e2-a0f6-254751460c78)

How do I have to change the code so that only the element
"Neuigkeit" is displayed. These values come from the managed property "owstaxIdNewsKategorie"

Where are there explanations on how to better display these managed properties in self-created layouts?

**Comment by kasperbolarsen** [comment]
The usual approach is to map the crawled property behind owstaxIdNewsKategorie to a RefinableString. This will ensure that only the label value/text is displayed
